pygmt/tests/test_rose.py

```diff
@@ -152,7 +152,7 @@ def test_rose_no_sectors(data_fractures_compilation):
         region=[0, 500, 0, 360],
         diameter="10c",
         labels="180/0/90/270",
-        frame=["xg100", "yg45", "+t'Windrose diagram'"],
+        frame=["xg100", "yg45", "+tWindrose diagram"],
```

Comment:
Does `"+t'Windrose diagram'"` still work or not?

Reply:
Yes it does still work, but the single quotes would be printed. I think it was a typo from whoever wrote that test.